internal/core/src/segcore/collection_c.cpp

```diff
@@ -24,6 +29,9 @@ void
 DeleteCollection(CCollection collection) {
     auto col = (milvus::segcore::Collection*)collection;
     delete col;
+#ifndef __APPLE__
```

Comment:
malloc_trim() should be enabled on linux

Reply:
Right. It is only disabled on MacOS. (ifndef)